usr/src/lib/libnvpair/mapfile-vers

```diff
@@ -198,6 +198,7 @@ SYMBOL_VERSION SUNW_1.1 {
 	nvlist_alloc;
 	nvlist_dup;
 	nvlist_free;
+	nvlist_invalidate;
```

Comment:
You can't add the symbols to already existing public version, either move it to SUNWprivate_1.1, or provide new public version.